strawberry_django/type.py

```diff
@@ -79,9 +80,29 @@ def _process_type(
     select_related: Optional[TypeOrSequence[str]] = None,
     prefetch_related: Optional[TypeOrSequence[PrefetchType]] = None,
     disable_optimization: bool = False,
+    fields: Optional[Union[List[str], str]] = None,
+    exclude: Optional[List[str]] = None,
     **kwargs,
 ) -> _O:
     is_input = kwargs.get("is_input", False)
+
+    if not exclude:
+        exclude = []
+
```

Comment:
You don't actually need this because you `isinstance` should be enough protection